_config.yml

```
@@ -29,8 +29,9 @@ permalink: pretty
 subtitle: Email government officials and council members to reallocate egregious police
   budgets towards education, social services, and dismantling racial injustice via
   this crowd-sourced tool.
-meta: Email government officials and council members to reallocate police budgets
-  towards education, social services, and dismantling racial injustice.
+# On Facebook the preview card only shows the first ~75 chars, so keep it short
+meta: Send pre-populated localized emails to officials in your city.
+meta_localized: Send pre-populated localized emails to officials in {{ page.title }}.
```

Ooooh great catch on the short character limit for FB!

“Send pre-populated localized emails to officials” feels a little wordy/overly technical for a mass audience. Something more like this?

```suggestion
  meta: Send a pre-written email with your city’s specific budget concerns.
  meta_localized: Send a pre-written email to directly to {{ page.title }} officials.
```

Simplified from Grade 10 to Grade 8 [via Hemingway](http://www.hemingwayapp.com/)~

---

Nice! Better.